src/config/logger/index.ts

```diff
@@ -0,0 +1,25 @@
+import { $log, PlatformLoggerSettings } from '@tsed/common';
+
+import { isProduction } from '../envs/index';
+
+if (isProduction) {
```

Comment:
convenient to have in dev to, imho

Reply:
sure, this was added by tsed cli - not sure what it does yet lol